homeworks/andrii.melnyk_melnyksan/homework_2/task-2a_pyramid__20181109.js

```diff
@@ -0,0 +1,11 @@
+const maxValue = 15;
+let strSpace;
+let strHash;
+
+for (let i = 0; i < maxValue; i++) {
+    strSpace = '';
+    strHash = '###';
+    for (let j = 0; j < maxValue - i; j++) strSpace += ' ';
+    for (let j = 0; j < 2 * i + 1; j++) strHash += '#';
+    console.log(strSpace + strHash);
+}
```

Comment:
working solution, but why do you have 3 loops where you could have only 1?

try to use .repeat() method for strings

Reply:
Thanks for your feedback!

I know that it maybe isn't the best way how to do it,  isn't the most 'proper' in this situation. 
But at current moment I'm just learning how to do it, how to do it in different ways, how to do it better, what is 'better practice' for this case. 
As I am a beginner in JavaScript development it exist many unknown things that I need to deal with, exept JS is also Ubuntu, vs code, nodejs and  their environment, and else many interesting stuff. So not everything goes so fast as I want, but it is only the beginning)

At that moment when I find solution and I'm trying to understand how it works and how I can... could I use it in different cases. 
So in this case I image myself that 
- using 1st loop I defines number of rows
- using 2nd and 3rd loops I form groups of elements in each rows, separately for spaces and hashes, and than concatenate it together. 

When I solved this task this approach seemed more clear, and I thought that I would be able to use it in many others cases, so I used it in this case.

Next time I try to use .repeat() method, thanks.

Reply:
It's absolutely fine, don't take as abuse or offence :) Programming is about solving issues in the most efficient way, not creating other issues. We just try to teach our students to keep up with this philosophy and using best practices ;)

But for now you need to fix:
1. remove package.json (for this homework you need only files with tasks)

Thanks!